submissions/irynatsarenok/pop_up/pop_up.html

```diff
@@ -0,0 +1,240 @@
+              Photos
+            </a>
+            <a href="#" class="popup_link link">
+              <img
+                src="pop_up_icons/apps.png"
+                alt="Apps"
+                width="55"
+                height="40"
+                class="popup_img"
+              />
+              Apps
+            </a>
+            <a href="#" class="popup_link link">
+              <img
+                src="pop_up_icons/documents.png"
+                alt="Documents"
+                width="55"
+                height="40"
+                class="popup_img"
+              />
+              Documents
+            </a>
+          </div>
+
+          <input type="checkbox" id="checkbox_more" class="checkbox_more" />
+          <label for="checkbox_more" class="popup_label_more">More</label>
+
+          <div class="popup_more">
+            <div class="popup_div">
+              <a href="#" class="popup_link link">
+                <img
+                  src="pop_up_icons/chat.png"
+                  alt="Chat"
+                  width="55"
+                  height="40"
+                  class="popup_img"
+                />
+                Chat
+              </a>
+              <a href="#" class="popup_link link">
+                <img
+                  src="pop_up_icons/classroom.png"
+                  alt="Classroom"
+                  width="55"
+                  height="40"
+                  class="popup_img"
+                />
+                Classroom
+              </a>
+              <a href="#" class="popup_link link">
+                <img
+                  src="pop_up_icons/collection.png"
+                  alt="Collection"
+                  width="55"
+                  height="40"
+                  class="popup_img"
+                />
+                Collection
+              </a>
+            </div>
+            <div class="popup_div">
+              <a href="#" class="popup_link link">
+                <img
+                  src="pop_up_icons/mars.png"
+                  alt="Mars"
+                  width="55"
+                  height="40"
+                  class="popup_img"
+                />
+                Mars
+              </a>
+              <a href="#" class="popup_link link">
+                <img
+                  src="pop_up_icons/blogs.png"
+                  alt="Blogs"
+                  width="55"
+                  height="40"
+                  class="popup_img"
+                />
+                Blogs
+              </a>
+              <a href="#" class="popup_link link">
+                <img
+                  src="pop_up_icons/promo.png"
+                  alt="Promo"
+                  width="55"
+                  height="40"
+                  class="popup_img"
+                />
+                Promo
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <a><img src="pop_up_icons/bell.png" alt="" width="25" height="25" /></a>
```

Comment:
With the missing `href` attribute browser see this element like `span` not like a link.